src/lib/util/validation.js

```diff
@@ -2,6 +2,7 @@ import { set, parsePath } from "/vendor/@10xjs/form";
 
 export const REQUIRED = "VALIDATION_REQUIRED";
 export const UNIQUE_CONSTRAINT = "VALIDATION_UNIQUE_CONSTRAINT";
+export const INVALID_SYMBOL = "INVALID_SYMBOL";
```

Comment:
I previously decided that all validation error types would be prefixed with `VALIDATION_`- although we haven't had enough use yet to prove that it's a useful pattern  🤔